.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+python:
+- "3.7"
```

Comment:
Python3.7 is not yet supposed by most build tools because it is pretty recent.
Please change this to a stable version of Python.